cpp/include/logistic_regression.h

```diff
@@ -64,38 +64,38 @@ class LogisticRegression {
     return predictions;
   }
 
+  /// Calculates the hypothesis of a given input vector
```

Comment:
Please add comments like \param, \return to doxygen comments. You can refer to cpu_operations.h and http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html for more information